src/inspector_agent.cc

```diff
@@ -779,7 +797,8 @@ void Agent::WaitForDisconnect() {
   }
   // TODO(addaleax): Maybe this should use an at-exit hook for the Environment
   // or something similar?
```

Comment:
It still feels like something like this would be the easier thing to do? That way the `contextDestroyed()` call would only be made *after* inspector sessions have disconnected?

Reply:
There are some clients who uses executionContextDestroyed as a signal that it is time to disconnect. Otherwise they do not have a way to detect that node process finished.

Reply:
In that case, can you remove the todo comment here? :)

Reply:
removed!